components/managed-identity/managed-identity.tf

```diff
@@ -109,14 +111,28 @@ data "azurerm_resource_group" "platform-rg" {
   name = "cft-platform-${local.acme_environment_rg}-rg"
 }
 
+data "azurerm_resource_group" "cftapps-mi-rg" {
+  name = "managed-identities-${local.wi_app_rg}-rg"
+}
+
 data "azurerm_key_vault" "acme" {
   name                = "acme${local.department_name}${local.acme_environment_app}${local.acme_environment_kv}"
   resource_group_name = data.azurerm_resource_group.platform-rg.name
 }
+
+resource "azurerm_user_assigned_identity" "wi-admin-mi" {
+  resource_group_name = data.azurerm_resource_group.cftapps-mi-rg.name
+  location            = data.azurerm_resource_group.cftapps-mi-rg.location
+
+  name = "admin-${local.wi_app_name}-mi"
+  tags = module.ctags.common_tags
+}
+
 resource "azurerm_role_assignment" "acme-vault-access" {
+  count                = length([azurerm_user_assigned_identity.sops-mi, azurerm_user_assigned_identity.wi-admin-mi])
```

Comment:
count should be avoided by default due to a number of issue with it, you should use `for_each` instead